### PR TITLE
Archana_Assignment_23_Done_2Hrs

### PR DESCRIPTION
**Pull request by ArchanaNJun20**
ArrayUnion:
image
ArrayIntersection:
image

### CODE REVIEW

File: src/archanaNJun20/ArrayIntersection.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package archanaNJun20;
+
+public class ArrayIntersection {
+	double[] getIntersectionOfArray(double[] array1, double[] array2){		
+		int  arraycount=0;
+		int length=0;
+		if(array1.length>array2.length)
+			length= array1.length;
```
Comment (KrishnaKTechnocredits) [inline src/archanaNJun20/ArrayIntersection.java:8]: For intersection of an array, you should create array length of min array. You have written logic for max array. Change it.

File: src/archanaNJun20/ArrayUnion.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package archanaNJun20;
+
+public class ArrayUnion {
+	int arraycount=0;
+	double[] getUnionOfArray(double[] array1,double[] array2){
+		int arraycount=0;
```
Comment (KrishnaKTechnocredits) [inline src/archanaNJun20/ArrayUnion.java:5]: Why you have created two variable with same name? Any specific reason?

Comment (ArchanaNJun20) [reply src/archanaNJun20/ArrayUnion.java:5]: sorry, my bad.by mistake I declare it twice.

File: src/archanaNJun20/ArrayIntersection.java (hunk 1 above)
Comment (ArchanaNJun20) [reply src/archanaNJun20/ArrayIntersection.java:8]: The code is updated.
Thank you :)

**Pull request merged**